ui/imports/shared/popups/addaccount/stores/AddAccountStore.qml

```diff
@@ -272,15 +271,13 @@ BasePopupStore {
                     root.selectedOrigin.keyUid === Constants.appTranslatableConstants.addAccountLabelOptionAddWatchOnlyAcc) {
                 return valid &&
                         !!root.watchOnlyAccAddress &&
-                        root.watchOnlyAccAddress.alreadyCreatedChecked &&
```

Comment:
this is needed!!

Reply:
Then, since this is `false` by default, the "Add account" button will be disabled